e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,44 @@
+const { createTestSubaccount } = require("../utils/account");
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+    beforeAll(async () => {
+        testAccount = await createTestSubaccount();
+    });
+
+    afterAll(async () => {
+        await browser.close();
+        await testAccount.delete();
+    });
+
+    it("navigates to seed phrase page successfully", async () => {
+        const user = await page.goto("/").then(() => page.getDocument());
+        await user
+            .findByRole("button", {
+                name: "Import Existing Account",
+            })
+            .then((importAccountBtn) => importAccountBtn.click())
+            .then(() => user.findByText("Recover Account"))
+            .then((recoverAccountBtn) => recoverAccountBtn.click());
+
+        expect(page).toMatchURL(/\/recover-seed-phrase$/);
+    });
+
+    it("recovers the account successfully", async () => {
+        const user = await page.getDocument();
+        await user
+            .findByRole("textbox")
+            .then((seedphraseInput) =>
+                seedphraseInput.type(testAccount.seedPhrase)
+            )
+            .then(() => user.findByRole("button"))
+            .then((btn) => btn.click())
+            .then(() => page.waitForNavigation());
+
+        expect(page).toMatchURL(/.org\/$/);
+        // TODO use data-testid prop
```

Comment:
Hah! Great minds think alike!

Can you add `data-test-id` to these elements and use them in this PR?

I think as we go along, we're going to need to make `adding data-test-id` to things we need to interact with during `e2e` tests `step 1` of testing any particular page

Reply:
In this case I agree since this particular element reflects current logged in state and which user is logged in. I don't prefer using the CSS class selector for tests here and in general 😄